fern/src/vm.rs

```diff
@@ -140,14 +142,17 @@ impl VM {
         Ok(())
     }
 
+    fn decode_op(&self) -> Op {
+        Op::decode_packed(self.code[self.ip]).unwrap()
+    }
```

Comment:
imo we should be returning a result here, see my comment above but of course open to discussion

Reply:
How would this ever fail?

Reply:
I kinda assumed op decoding was fallible given it returns an option. Is it not possible to try to load an invalid Op, say if we're loading from a file or something?

Reply:
Actually given my comment above this is probably okay like it shouldn't matter since we're the only "users" of this "library"